mscore/ove.cpp

```diff
@@ -5183,7 +5183,7 @@ int getInt(int byte, int bits) {
       int num = 0;
 
       if( bits > 0 ) {
-            int factor = int(pow(2.0, bits-1));
+            int factor = 1 << (bits - 1);
```

Comment:
Could you please leave a comment that this operation equals to the `pow(2,bits-1)`? Same above

Reply:
And force push the changes! Let's test the fix proposed by the Travis team and applied by me 2 mins ago :)

Reply:
Is it really? What if bits is 2? 2-1 is 1, square is still one, but shift 1 is 2

Reply:
`pow(2, 1)` is of course 2...

Reply:
@anatoly-os Done!